sycl/source/detail/jit_compiler.hpp

```diff
@@ -98,6 +98,7 @@ class jit_compiler {
   CompileSYCLFuncT CompileSYCLHandle = nullptr;
   ResetConfigFuncT ResetConfigHandle = nullptr;
   AddToConfigFuncT AddToConfigHandle = nullptr;
+  void *MLibraryHandle = nullptr;
```

Comment:
Could this be a unique pointer as well, reusing the custom deleter from the newly-added unique pointer in `jit_compiler`'s constructor? Then we wouldn't need to duplicate the unload logic in the destructor.

Reply:
Nit: The other members here don't carry the `M` prefix.

Reply:
done in https://github.com/intel/llvm/pull/16517/commits/c86ef55dff0030f37b4f93f1c3f4ed62d6466c39

Reply:
title is slightly updated too.